website/chimera.php

```diff
@@ -0,0 +1,86 @@
+<?
+	include ("include.php");
+	
+	if (isset($_GET['type']))
+		$type = $_GET['type'];
+	else
+		$type = "saia_chimr";
+		
+	if (isset($_GET['indexnum']))
+		$indexnum = $_GET['indexnum'];
+	else
+		$indexnum = "1";

```

Comment:
I would have remove the `else` in these statments by  defining: `type` and `idnexnum` first and having only an `if` for each, without the `else`. But that's just me being minimalist.